Source/AudioClockDirector.swift

```diff
@@ -113,3 +124,9 @@ extension AudioClockDirector {
         bufferClosures.broadcast(key: key, payload: buffered)
     }
 }
+
+extension AudioClockDirector {
+    func changeInQueue(_ key: Key, url: URL) {
+        audioQueueClosures.broadcast(key: key, payload: url)
```

Comment:
See comment below. I would use the AudioQueueDirector for all the broadcasting rather than the AudioClockDirector.